data/transport missions.txt

```diff
@@ -3404,3 +3404,68 @@ mission "Capture Smuggler"
 		payment 150000
 		conversation
 			`The huge man drags the unconscious smuggler out of the cargo container with one hand and carries the cargo box with the other. He stops in front of you. "<first> <last>," he says. "My employer, he will be very happy. Your payment, you will receive soon. Is it not good, to be reliable?" The man leaves your ship, letting the smuggler's legs scrape along the ground as he walks. A moment later, you receive <payment> from an anonymous account.`
+
+
+
+mission "Inheritance Redirection"
+	description "Travel to <destination> to act as heir for the late warlord Limping Pappa."
+	source
+		government "Pirate"
```

Comment:
Does this have to only offer on pirate worlds? How about instead having it offer on frontier worlds. (I gave all pirate worlds the frontier attributes some time ago so only specifying frontier worlds means that it will still offer on pirate worlds as well.)

Reply:
Frontier worlds sounds quite appropriate. Changed to frontier worlds.